client/my-sites/domains/domain-management/list/domain-item.jsx

```diff
@@ -383,4 +388,21 @@ class DomainItem extends PureComponent {
 	}
 }
 
-export default localize( DomainItem );
+const addEmailClick = ( domain ) =>
+	composeAnalytics(
+		recordGoogleEvent(
```

Comment:
We should check if we need to send those events to Google though. Not sure what's the use case of having both tracks and GA events.